storey/table.py

```diff
@@ -525,12 +525,12 @@ def calculate_features(self, timestamp, windows):
 
             for bucket_index in range(current_time_bucket_index, last_bucket_to_aggregate - 1, -1):
                 if bucket_index < len(self.buckets):
-                    t, v = self.buckets[bucket_index].get_value()
-                    aggregated_value.aggregate(t, v)
+                    bucket = self.buckets[bucket_index]
+                    aggregated_value.aggregate(bucket.last_time, bucket.value)
```

Comment:
what if self.aggregation == first? in this case get_value() will return first_time and not last_time

Reply:
True... Kind of disturbing that no existing test caught that.